src/main/php/PHPMD/TextUI/XdebugOptionHandler.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace PHPMD\TextUI;
+
+use Composer\XdebugHandler\XdebugHandler;
+
+/**
+ * Xdebug CLI Option Handler
+ *
+ * Enables instead of disables Xdebug, if called with "--xdebug" CLI option.
+ */
+class XdebugOptionHandler extends XdebugHandler
+{
+    /**
+     * Rebuilds the run command with Xdebug enabled, instead, if CLI option "--xdebug" is used
+     */
+    protected function restart(array $command): void
+    {
+        if (in_array('--xdebug', $command, true)) {
+            // Unset unwanted command arguments & options
+            if (($xdebugKey = array_search('--xdebug', $command, true)) !== false) {
+                unset($command[$xdebugKey]);
+            }
+            if (($noConfigKey = array_search('-n', $command, true)) !== false) {
+                unset($command[$noConfigKey]);
+            }
+            if (($configKey = array_search('-c', $command, true)) !== false) {
+                unset($command[$configKey + 1]);
+                unset($command[$configKey]);
```

Comment:
```suggestion
                unset($command[$configKey + 1], $command[$configKey]);
```

Reply:
I find it a tad bit more readable like before.

Or how about this?

```suggestion
                unset(
                    $command[$configKey + 1],
                    $command[$configKey]
                );
```